.github/workflows/build.yml

```diff
@@ -43,28 +43,28 @@ jobs:
       uses: actions/setup-python@v1
 
     - name: Setup Toolchain
-      uses: fiam/arm-none-eabi-gcc@v1.0.2
+      uses: fiam/arm-none-eabi-gcc@master
```

Comment:
Just a question, why fiam?

Reply:
is there another action source that does the same?